docs/context/ref/ChatInitSettings.md

```diff
@@ -0,0 +1,86 @@
+---
+id: ChatInitSettings
```

Comment:
Suggest calling this ChatSettings?   It seems like these settings would be relevant even after the chat had been created, and could be the basis for further, ongoing intents.  e.g. EmailTranscript, CreateAppointment, RecordInCRM, etc. which could be handled by context listeners.